syntax = "proto3";
package example;

// The machine learning service definition.
service MachineLearning {
  // Sends a machine learning result
  rpc StartLearn (SendRequest) returns (GetReply) {}
}

// The request message containing the params.
message SendRequest {
  string name = 1;
}

// The response message containing the results
message GetReply {
  string message = 1;
}